gradle/groovyProject.gradle

```diff
@@ -46,7 +46,7 @@ testTasks.all { task ->
 jarTasks.all { jar ->
     jar.manifest.mainAttributes(
         (Attributes.Name.IMPLEMENTATION_TITLE.toString()): 'Gradle',
-        (Attributes.Name.IMPLEMENTATION_VERSION.toString()): version,
+        (Attributes.Name.IMPLEMENTATION_VERSION.toString()): versionBase
```

Comment:
I think the plugin author could try to load the manifest file of the jar - but I don't see any reason why he would do that. He could just use `GradleVersion` for that.